test-project/helloFX/src/main/java/com/gluonhq/substrate/test/Main.java

```diff
@@ -26,6 +26,9 @@ public void start(Stage stage) {
 
     public static void main(String[] args) {
         System.setProperty("prism.verbose", "true");
+        if (System.getProperty("javafx.platform") == null) {
```

Comment:
We don't really need to hardcode the prism pipeline that we want use. javafx will automatically fallback to the software pipeline when the hardware pipeline fails to start. And in the unlikely case that hardware support on CI systems becomes available in the future, we'll use it for free.

Reply:
Makes sense, pushed.